### PR TITLE
Added methods to find linewidth maps

### PR DESCRIPTION
**Pull request by astrofrog**
As discussed with @keflavich

### CODE REVIEW

**Comment by astrofrog** [comment]
@keflavich - ready for review!

**Comment by astrofrog** [comment]
Where does the changelog entry go? Do I make a new 0.4.0 section?

**Comment by keflavich** [comment]
@astrofrog Yeah, new section is needed.  0.4 sounds good to me, since this is an API change and the 0.3.0->0.3.1 was only bugfixes

File: spectral_cube/spectral_cube.py [hunk 1]
```diff
@@ -1481,17 +1504,38 @@ def moment0(self, axis=0, how='auto'):
...
         """
         return self.moment(axis=axis, order=2, how=how)
 
+    def linewidth_sigma(self, how='auto'):
+        """
+        Compute a (sigma) linewidth map along the spectral axis.
+
+        For an explanation of the ``how`` parameter, see :meth:`moment`.
+        """
+        with np.errstate(invalid='ignore'):
```
Comment (keflavich) [inline spectral_cube/spectral_cube.py]: we need this in many more locations in SpectralCube

**Comment by keflavich** [comment]
:+1:

**Comment by astrofrog** [comment]
@keflavich - I've added the changelog entry

**Comment by astrofrog** [comment]
@keflavich - good to merge?

**Pull request merged**